notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -32,9 +32,10 @@
     "from utilities.plotting import (\n",
     "    plot_projection, plot_regression, check_mirrors, get_cmaps, table_from_dict\n",
     ")\n",
-    "from utilities.kernels import linear_kernel, gaussian_kernel, center_kernel\n",
-    "from utilities.classes import KPCA, KRR, SparseKPCA, SparseKRR\n",
-    "from utilities.classes import KPCovR, SparseKPCovR\n",
+    "from utilities.kernels import linear_kernel, gaussian_kernel, center_kernel # to be replaced with functions from skcosmo\n",
```

Comment:
You can delete this if it is not used elsewhere.

Reply:
They are used somewhere.

Reply:
Please adapt this like you did in #29. There shouldn't be any kernel-tutorials kernel functions in this notebook. You should use sklearn kernels and centerers from skcosmo (https://github.com/cosmo-epfl/sklearn-cosmo/pull/8)

Reply:
Okay.